### PR TITLE
Add cloc

### PR DESCRIPTION
**Pull request by sbrl**
Find out more about cloc here: https://github.com/AlDanial/cloc

### CODE REVIEW

**Comment by waldyrious** [comment]
LGTM, but do you think the "base" is really necessary in the token paths? I would suggest it to be removed.

File: pages/common/cloc.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> Count, and compute differences of, lines of source code and comments.
+
+- Count all the lines of code in a directory:
+
+`cloc {{/path/to/base/directory}}`
+
+- Count all the lines of code in a directory, displaying a progress bar during the counting process:
+
+`cloc --progress 1 {{/path/to/base/directory}}`
```
Comment (waldyrious) [inline pages/common/cloc.md]: what does the 1 mean here?

**Comment by sbrl** [comment]
@waldyrious On second thought, removing the base path would probably be a good idea.

The `1` is the value of the `--progress` flag. It tells `cloc` that it should output it's progress on every file. Setting it to `2` would cause `cloc` to output it's progress every 2 files.

**Comment by waldyrious** [comment]
Shouldn't it be `--progress-rate=N`, per the [documentation](https://github.com/AlDanial/cloc#options-)?

**Comment by sbrl** [comment]
@waldyrious Oh, right! I didn't notice. It still seems to work either way though. I'll update the documentation anyway.

**Comment by waldyrious** [comment]
I think the long version makes it clearer why the 1 is needed :) LGTM 👍

File: pages/common/cloc.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# cloc
+
+> Count, and compute differences of, lines of source code and comments.
+
+- Count all the lines of code in a directory:
+
+`cloc {{directory}}`
```
Comment (waldyrious) [inline pages/common/cloc.md]: Sorry if I wasn't clear before. I meant we should leave this as just `{{/path/to/directory}}`.

Comment (sbrl) [inline pages/common/cloc.md]: Oh right! Is that better?

**Comment by waldyrious** [comment]
Almost there! I'd suggest either .../directory/one or .../first/directory (and the equivalent for the 2nd).

**Comment by sbrl** [comment]
How about that?

**Comment by waldyrious** [comment]
perfect 👍 LGTM

**Comment by sbrl** [comment]
:D

**Pull request merged**